crates/sui-framework/sources/crypto/ecvrf.spec.move

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+spec sui::ecvrf {
+    spec ecvrf_verify {
+        // TODO: temporary mockup.
```

Comment:
what is this for?

Reply:
I'm not sure. It seems like the other crypto modules have it. I'll remove it for now.